examples/plot_kmean_poincare_disk.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
filename should be plot_kmeans_poincare_disk.py as we decided on `kmeans` as the naming convention